backend/endpoints/rom.py

```diff
@@ -214,11 +225,27 @@ async def get_rom_content(
     files_to_download = files or [r["filename"] for r in rom.files]
 
     if not rom.multi:
-        return FileResponse(path=rom_path, filename=rom.file_name)
+        return FileResponse(
+            path=rom_path,
+            filename=rom.file_name,
```

Comment:
I found that using `FileResponse` when trying to use X-Accel, makes Gunicorn still use a lot of CPU time, and doesn't completely delegate the download to nginx. Download speeds I was able to measure were also slower when using `FileResponse`.